### PR TITLE
Docs: Add Full Page HTML guide.

### PR DESCRIPTION
**Pull request by gorzelinski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Add Full Page HTML guide. Closes https://github.com/cksource/ckeditor5-internal/issues/3133.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

**Review by AnnaTomanek — CHANGES_REQUESTED** [review]

File: packages/ckeditor5-html-support/docs/features/full-page-html.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+category: features-html
+order: 50
+modified_at: 2023-05-15
+---
+
+# Full page HTML
+
+{@snippet features/general-html-support-source}
+
+The Full Page HTML feature allows you to preserve the whole HTML page in the editor data. With this plugin, you can enable certain HTML tags in your editor. But, unlike GHS, which focuses on tags inside the content, this feature keeps the markup that's usually not visible to the user, like the wrapping `<html></html>` tag.
```
Comment (AnnaTomanek) [inline packages/ckeditor5-html-support/docs/features/full-page-html.md:11]:
```suggestion
Thanks to the full page HTML feature you can use CKEditor 5 to edit entire HTML pages (from `<html>` to `</html>`), including the page metadata. While the {@link features/general-html-support General HTML Support} feature focuses on elements inside the content (the document's `<body>`), this feature enables markup mostly invisible to the end user.
```
What do you think?

File: packages/ckeditor5-html-support/docs/features/full-page-html.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	.catch( /* ... */ );
+```
+
+<info-box info>
+	Read more about {@link installation/plugins/installing-plugins installing plugins}.
+</info-box>
+
+## Additional feature information
+
+Here are some examples of tags you can enable by using this plugin:
```
Comment (AnnaTomanek) [inline packages/ckeditor5-html-support/docs/features/full-page-html.md:46]:
```suggestion
Here are some examples of the HTML elements you can enable with this plugin:
```

File: packages/ckeditor5-html-support/docs/features/full-page-html.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+* The HTML doctype preamble: `<!DOCTYPE html>`
+* The HTML root element: `<html></html>`
+* The head element: `<head></head>`
+* Document's title: `<title></title>`
```
Comment (AnnaTomanek) [inline packages/ckeditor5-html-support/docs/features/full-page-html.md:51]:
```suggestion
* The document's title: `<title></title>`
```

File: packages/ckeditor5-html-support/docs/features/full-page-html.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+* Meta tags inside the head element like: `<meta name="description" content="Meta description">`
```
Comment (AnnaTomanek) [inline packages/ckeditor5-html-support/docs/features/full-page-html.md:52]:
```suggestion
* Meta tags inside the head element, like: `<meta name="description" content="Meta description">`
```

**Review by AnnaTomanek — APPROVED** [review]
I added a few suggestions, check them out. Also cleaned up the Related features sections in a few guides.
Leftovers:
* Changing the HTML elements in the list to just use the opening tags.
* To consider: using CKBox instead of EasyImage for image upload to be consistent with other demos.

Overall a really nice guide, and the example is great 🤩 Thanks!

File: packages/ckeditor5-html-support/docs/features/full-page-html.md (hunk 1 above)
Comment (gorzelinski) [reply packages/ckeditor5-html-support/docs/features/full-page-html.md:11]: Nicely rephrased - the description is more clear now.

File: packages/ckeditor5-html-support/docs/features/full-page-html.md (hunk 3 above)
Comment (apadol-cksource) [reply packages/ckeditor5-html-support/docs/features/full-page-html.md:51]: We don't need to use closing tags when writing about HTML elements

**Pull request merged**